guides/account_delegation.md

```diff
@@ -0,0 +1,81 @@
+# Account Delegation
+
+## What is account delegation
+
+Account delegation provides access and/or control over your private encrypted data to another user. The intended purpose for this feature is to allow people you trust full access. In almost all cases we recommend using Shares and On-Shares to give access to other users as it gives much more granular control over the data shared to another user. An example where you might use a feature like this is where a user is helping a family member (be that a child or an elderly parent) who requires assistance.
+
+## Setting up delegation
+
+### Setting up the Scenario
+
+First lets create two new users to set up a delegation between.
+
+```bash
+meeco users:create -p password > .riker.yaml
+meeco users:create -p password > .homer.yaml
+```
+
+Next lets create an item in Riker's account so we can later test to make sure Homer has access to it.
+
+```bash
+# List out available templates
+meeco templates:list -a .riker.yaml
+# Create an item config file from one of the chosen templates
+meeco items:create-config vehicle -a .riker.yaml > .riker-vehicle-config.yaml
+# Extra step, open up the above item config yaml file and modify it in your chosen text editor
+# Create the item using the above item config yaml
+meeco items:create -i .riker-vehicle-config.yaml -a .riker.yaml > .riker-vehicle.yaml
+```
+
+### Creating the Delegation Connection
+
+Next lets get Riker to create a delegation connection invitation with the delegation role `reader`. (delegation-role options are `owner`, `admin`, and `reader`)
+
+```bash
+meeco delegations:create-invitation -a .riker.yaml Homer reader > .delegation-invitation.yaml
```

Comment:
I was testing against `stage` environment and `reader` role did not work. I assume fix is not deployed to `stage` environment yet? Everything works well if using `owner`instead.

Error with `reader` role:
```
Error: unexpected error: {"size":0,"timeout":0}
```

Reply:
Yea that is the bug I found in vault that Yuri fixed last week, to my knowledge it hasn't been rolled out to stage yet so this branch only works against dev.